benchmarks/source-agilitycms/gatsby-config.js

```diff
@@ -0,0 +1,65 @@
+require("dotenv").config({
+  path: `.env.${process.env.NODE_ENV}`,
+})
+
+//configure your agility plugin with environment variables so that
+//your agility api credentials stay secure
+const agilityConfig = {
+  guid: process.env.AGILITY_GUID,
```

Comment:
Can you please prefix all environment variables with `BENCHMARK_`? This helps us easily identify benchmark related environment variables and conforms with the other benchmarks, ex. https://github.com/gatsbyjs/gatsby/blob/947492971d33a508ace63f24c0be20870f3ae892/benchmarks/source-kontent/gatsby-config.js#L17